backend/src/models/User.js

```diff
@@ -5,16 +5,16 @@ const userSchema = new mongoose.Schema(
         email: {
```

Comment:
This schema is for admin user right, can we rename it to Admin or AdminUser instead

Reply:
I have updated the name to Admin

Reply:
I don't see it, we should change the filename as well to Admin.js or so